fundamentals/script.js

```diff
@@ -0,0 +1,216 @@
+const TEST_DATA = {
+  ARRAY_OF_INTS: [1, 5, 6, 7]
+}
+
+var student = {
+  NAME : 'David Rayy',
+  SCLAS : 'VI',
+  ROLLNO : 12
+};
+
+var stringToFormat = ' asd asd jjj bbb jjj '; // Global on purpose
+
+/**
+ * TASK 1
+ *
+ */
+function getBiggerNumber(int1, int2) {
+  if ((int1 >= 40 && int1 <= 60) && (int2 >= 40 && int2 <= 60)) {
+    return int1 > int2 ? int1 : int2;
+  }
+
+  return 0;
+}
+
+/**
+ * TASK 2
+ *
+ */
+function getArraySum(intArr) {
+  return intArr.reduce(function (accumulator, currentValue) {
+    return accumulator += currentValue;
+  }, 0);
+}
+
+/**
+ * TASK 3
+ *
+ */
+function colorMixer() {
+  myColorsArray = ['Red', 'Green', 'White', 'Black'];
+  const output = myColor.join(',');
+}
+
+/**
+ * TASK 4
+ *
+ */
+function listProperties() {
+  let returnString = '';
+  for (let key in student) {
+    if (student.hasOwnProperty(key)) {
+      returnString += key + ',';
+    }
+  }
+}
+
+
+/**
+ * TASK 5
+ *
+ */
+function stripLeadTrail() {
+  stringToFormat.trim();
+}
+
+/**
+ * TASK 6
+ *
+ */
+function removeFirst() {
+  stringToFormat.replace('jjj','');
+}
+
+/**
+ * TASK 7
+ *
+ */
+function removeFirst() {
+  stringToFormat.search('jjj');
+}
+
+/**
+ * TASK 8
+ * '<p><strong><em>PHP Exercises</em></strong></p>' -- test string
+ */
+function strip_html_tags(string) {
+  const regExHTML = /<[^>]*>/g;
+  return string.replace(regExHTML, '');
+}
+
+/**
+ * TASK 9
+ * 'Hello world, welcome to the universe.' -- test string
+ */
+function checkIfEnds(inputString, end) {
+  return inputString.endsWith(end);
+}
+
+/**
+ * TASK 10
+ *
+ */
+function objectLenght(inputObject) {
+  return Object.keys(inputObject).length;
+}
+
+/**
+ * TASK 11
+ *
+ */
+function first(array, n) {
+  if(array == NULL){
+    return;
+  }
+  if (n <= 0) {
+    return array[0];
+  }else{
+    return array.splice(n, array.length-1);
+  }
+}
+
+/**
+ * TASK 12
+ *
+ */
+function removeFirst() {
+  const arr1=[3, 'a', 'a', 'a', 2, 3, 'a', 3, 'a', 2, 4, 9, 3];
+  let itterObject = {};
+  let result = '';
+
+  arr1.forEach(element => {
+    if (itterObject.hasOwnProperty(element)) {
+      itterObject[element] += 1;
+    } else {
+      itterObject[element] = 1;
+    }
+  });
+
+  Object.keys(itterObject).forEach(key => {
+    let currentValue = itterObject[key];
+
+    if (currentValue > maxCount) {
+      maxCount = currentValue;
+      result = key + ': ' + currentValue;
+    }
+  });;
+  return result;
+}
+
+/**
+ * TASK 13
+ *
+ */
+function fizzBuzz() {
+  let outputString = '';
+
+  for (let i = 1; i <= 100; i++) {
+    if((i % 15) == 0) outputString += 'FizzBuzz';
```

Comment:
Please use the full syntax for if and else-ifs, this is the standardised way in javascript. This comment applies to the whole file, not only this line (just so I don't repeat this comment on other lines)